src/makeInfinityAdder.js

```diff
@@ -7,6 +7,21 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  return function adder(add = false) {
```

Comment:
It's not the best approach to assign the value to false as it could lead to confusion and unexpected behavior. You can check if it's undefined instead